src/components/assignment/js/defaults.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var env = {
+    account: '',
+    project: '',
+    group: '',
+    groupId: '',
+    token: '',
+    server: {
+        host: 'api.forio.com',
+        protocol: 'https'
+    }
+};
+
+module.exports = {
+    set: function (options) {
+        env = _.merge(env, options);
+    },
+
+    get: function () {
+        return env;
```

Comment:
This was pretty much what the `configuration service` was intended for. That service itself is pretty half-baked, but we should see if we can combine.